content/en/security/guide/aws_fargate_config_guide.md

```diff
@@ -11,6 +11,8 @@ further_reading:
 
 This guide walks you through configuring [Cloud Security Management (CSM)][3], [Application Security Management (ASM)][4], and [Cloud SIEM][5] on AWS Fargate.
 
+{{< img src="security/datadog_security_coverage_aws_fargate.png" alt="CSM graphic" width="90%">}}
```

Comment:
Can we get a more descriptive alt text? Maybe something like...
```suggestion
{{< img src="security/datadog_security_coverage_aws_fargate.png" alt="Flow chart showing how CSM, ASM, and Cloud SIEM are configured on AWS Fargate" width="90%">}}
```

Reply:
Whoops. I completely forgot to update that. Thanks for catching it.